packages/starboard-components/README.md

```diff
@@ -0,0 +1,35 @@
+## Installation
+
+*Future Requirement* Project package.json must specify if it is a shopify or wordpress project.
+```json
+{
+  "buildType": "shopify|wordpress",
```

Comment:
Just at first glance, the pipe between `shopify|wordpress` threw me a little bit – for the dummies like me who skim over docs, it might be better to only specify one version/value here.